### PR TITLE
Print info for parameters and images

### PR DESCRIPTION
**Pull request by alecomunian**
Hi Julien,
I did some very minor modifications to the code, just to print out some info about the simulation parameters and to the Img, that I found quite useful when debugging for the s2Dcd, but clearly feel free to accept them or not.
Bests,

### CODE REVIEW

**Review by julien-straubhaar — COMMENTED** [review]
Currently I use the version '0.31.3' of pyvista, and 'UniformGrid' object has no attribute 'cell_data'

**Comment by alecomunian** [comment]
Indeed, my PyVista version is 0.36.1.

File: .gitignore [hunk 1]
```diff
@@ -1 +1,12 @@
 .ipy*
+geone/__pycache__/
```
Comment (julien-straubhaar) [inline .gitignore:2]: These directories should not be here if geone is not installed "locally", but yes, we can add them in .gitignore...

File: geone/deesseinterface.py [hunk 2]
```diff
@@ -1676,6 +1676,27 @@ def __init__(self,
         self.seedIncrement = seedIncrement
         self.nrealization = nrealization
         self.ok = True # flag to "validate" the class
+    
+    def __str__(self):
```
Comment (julien-straubhaar) [inline geone/deesseinterface.py:1680]: Good idea to add __str__ function! I will add it in a next commit

File: geone/img.py [hunk 3]
```diff
@@ -82,6 +82,17 @@ def __init__(self,
     # ------------------------------------------------------------------------
     def __str__(self):
         """Returns name of the image: string representation of Image object"""
+        out = ("\n    "
```
Comment (julien-straubhaar) [inline geone/img.py:85]: Effectively, the output may be more explicit... I will add it

File: geone/imgplot3d.py [hunk 4]
```diff
@@ -325,7 +325,8 @@ def drawImage3D_surface (
 
     pg = pv.UniformGrid((xdim, ydim, zdim), (im.sx, im.sy, im.sz), (xmin, ymin, zmin))
 
-    pg.cell_arrays[im.varname[iv]] = zz #.flatten()
+    pg.cell_data[im.varname[iv]] = zz #.flatten()
```
Comment (julien-straubhaar) [inline geone/imgplot3d.py:330]: cell_data is not an attribute of UniformGrid in version 0.31.1 of pyvista...

**Pull request closed without merging**